src/modules/customer/containers/HappyUserPage/components/WizardForm/WizardFormPages/EighthPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form/immutable';
+import { FormattedMessage } from 'react-intl';
+
+import messages from '../messages';
+import formConstants from '../constants';
+
+const WizardFormEighthPage = props => {
+  const { handleSubmit, previousPage } = props;
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="wizard-form__container">
+        <label className="wizard-form__label">
+          {<FormattedMessage {...messages.notes} />}
+        </label>
+        <div className="wizard-form__container-input">
+          <Field
+            name={`${formConstants.formNamesNotes}`}
+            component="textarea"
+            className="wizard-form__textarea"
+          />
+        </div>
+      </div>
+      <div className="wizard-form__container">
+        <button
+          type="button"
+          className="wizard-form__btn-prev"
+          onClick={previousPage}
+        >
+          {<FormattedMessage {...messages.btnPrevious} />}
+        </button>
+        <button className="wizard-form__btn-submit" type="submit">
+          {<FormattedMessage {...messages.btnSubmit} />}
+        </button>
+      </div>
+    </form>
+  );
+};
+
+WizardFormEighthPage.propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  previousPage: PropTypes.func.isRequired,
+};
+
+export default reduxForm({
+  form: formConstants.FORM_NAME,
```

Comment:
you are going to have many forms in future. make them easy to understand, also by names:
`form: formConstants.HAPPY_USER_FORM,`